AmazonEmailServiceConfiguration.pas

```diff
@@ -0,0 +1,55 @@
+unit AmazonEmailServiceConfiguration;
+
+interface
+
+type
+  TAmazonEmailServiceConfiguration = class
+  private
+    function GetEnvVarValue(const VarName: string): string;
+  public
+    procedure GetFromEnvironment(var AwsSESEndpoint, AwsSESAccessKey, AwsSESSecretKey: string);
+  end;
+
+const
+  AWS_SES_REGION_ENDPOINT = 'AWS_SES_REGION_ENDPOINT';
```

Comment:
The configuration is `REGION`, not `REGION_ENDPOINT`. Remember, we need to follow the same standards as the official libraries.